test/e2e/BUILD

```diff
@@ -34,6 +34,7 @@ go_test(
         "//test/e2e/lifecycle:go_default_library",
         "//test/e2e/lifecycle/bootstrap:go_default_library",
         "//test/e2e/network:go_default_library",
+        "//test/e2e/network/netpol:go_default_library",
```

Comment:
hack/update-bazel.sh should remove this line (and therefore all the test/e2e/OWNERS dep

Reply:
done working now